@app/admin/next.config.js

```diff
@@ -2,7 +2,12 @@ const path = require('path');
 
 /** @type {import('next').NextConfig} */
 const nextConfig = {
-  transpilePackages: ['@cms/packages/ui', '@cms/packages/templates'],
+  transpilePackages: [
+    '@cms/ui',
```

Comment:
I only updated this import to demonstrate. You would want to do this for the rest of your untranspiled dependencies, too.

Reply:
Even if they are not components and are general modules used across the projects? Okay, makes sense.